c2corg_ui/templates/route/detailedattributes.html

```diff
@@ -0,0 +1,250 @@
+<%namespace file="../helpers/view.html" import="show_areas, show_attr"
+ />
+ 
+## LOCATION
+<%def name="get_route_location(route)">\
+
```

Comment:
Perhaps you could remove the unneeded blank lines at the beginning and end of the function?